app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -41,26 +55,36 @@ export default function SearchResult({ user }: { user: User.AsObject }) {
       <Card className={classes.card}>
         <CardActionArea>
           <CardContent>
-            <div className={classes.resultHeader}>
-              <Avatar user={user} />
-              <Container>
-                <Typography variant="h2">{user.name}</Typography>
-                <Typography variant="subtitle1">
+            <UserSummary user={user} />
+            <div className={classes.statusLabelWrapper}>
+              <div>
+                <CouchIcon />
+                <Typography
+                  className={classes.statusLabel}
+                  display="inline"
+                  variant="subtitle1"
+                  color="primary"
+                >
                   {hostingStatusLabels[user.hostingStatus]}
                 </Typography>
-              </Container>
+              </div>
+              <div>
+                <LocationIcon />
+                <Typography
+                  className={classes.statusLabel}
+                  display="inline"
+                  variant="subtitle1"
+                  color="secondary"
+                >
+                  {meetupStatusLabels[user.meetupStatus]}
+                </Typography>
+              </div>
             </div>
-
-            <ScoreBar value={user.communityStanding * 100}>
-              {COMMUNITY_STANDING}
-            </ScoreBar>
-            <ScoreBar value={user.verification * 100}>
-              {VERIFICATION_SCORE}
-            </ScoreBar>
-
-            {user.aboutMe.length < 300
-              ? user.aboutMe
-              : user.aboutMe.substring(0, 300) + "..."}
+            <Typography component="h3" variant="h6" className={classes.about}>
```

Comment:
just do `variant="body1"`

Reply:
Because I don't think this actually needs to be bigger, and also it isn't a heading

Reply:
I was thinking that soon after I left that comment, but didn't have time to write back/change until now

Reply:
great, done